test/unit/ngsiv2/examples/subscriptionRequests/errorNotification.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This error seems to be based in a NGSIv1 notification format... If the purpose is to have a JSON not following the NGSIv2 notification format, I'd suggest to use something more "wrongly", to avoid confusion.

Reply:
Is it possible that the Context Broker sends a wrong notification? Something similar to this example?

Reply:
Not intentionally... But if you are testing wrong notification, what I mean is to use something completly weird, eg:

```
{
   "foo": "A very wrongly formated NGSIv2 notification..."
}
```

Reply:
Or even a payload that not event follows JSON syntax, if you want to cover more alternatives.